.github/workflows/tests.yml

```diff
@@ -34,8 +34,10 @@ jobs:
     - name: Build
```

Comment:
Discussed offline: I have a somewhat strong preference for pushing as much of the testing, especially when ayb is a young project and subject to lots of internal refactors, into end-to-end testing. To that end, I think
* e2e tests should not be ignored --- they are the key determinant of correctness
* unless the confirm test is testing something the end-to-end test is not, the end-to-end test covers a superset of the code, and the duplicative confirm test should be removed

Reply:
I've decided to remove the unit tests, I'll mark this as resolved :)